src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -278,12 +281,18 @@ library BeaconChainProofs {
         require(proofs.timestampProof.length == 32 * (EXECUTION_PAYLOAD_HEADER_FIELD_TREE_HEIGHT),
             "BeaconChainProofs.verifyWithdrawalProofs: timestampProof has incorrect length");
 
+        // TODO: make historical proofs actually work. this is not compelete and does not function correctly right now
```

Comment:
are we fine merging with this?

Reply:
I'm OK merging into `newpodchanges` but I'm unclear on how this was supposed to be organized, really

Reply:
It was branched off of newpodchanges, which has some bug fixes + the new proofs.  If its alright with you, we merge this into newpodchanges, and then work off that branch.